src/utils/deep_research.py

```
@@ -0,0 +1,377 @@
+import pdb
```

Ruff (0.8.2)

`pdb` imported but unused

Remove unused import: `pdb`

(F401)